x-pack/filebeat/module/aws/vpc/config/input.yml

```diff
@@ -0,0 +1,50 @@
+{{ if eq .input "s3" }}
+
+type: s3
+queue_url: {{ .queue_url }}
+credential_profile_name: {{ .credential_profile_name }}
+
+{{ else if eq .input "file" }}
+
+type: log
+paths:
+  {{ range $i, $path := .paths }}
+  - {{$path}}
+    {{ end }}
+exclude_files: [".gz$"]
+
+{{ end }}
+
+processors:
+  - dissect:
+      field: message
+      target_prefix: aws.vpc
+      tokenizer: '%{version} %{account_id} %{interface_id} %{srcaddr} %{dstaddr} %{srcport} %{dstport} %{protocol} %{packets} %{bytes} %{start} %{end} %{action} %{log_status}'
+  - convert:
+      ignore_missing: true
+      fields:
+        - {from: aws.vpc.srcaddr, to: source.address}
+        - {from: aws.vpc.srcaddr, to: source.ip, type: ip}
+        - {from: aws.vpc.srcport, to: source.port, type: long}
+        - {from: aws.vpc.dstaddr, to: destination.address}
+        - {from: aws.vpc.dstaddr, to: destination.ip, type: ip}
+        - {from: aws.vpc.dstport, to: destination.port, type: long}
+        - {from: aws.vpc.protocol, to: network.iana_number, type: long}
+        - {from: aws.vpc.packets, to: source.packets, type: long}
+        - {from: aws.vpc.bytes, to: source.bytes, type: long}
+        - {from: aws.vpc.packets, to: network.packets, type: long}
+        - {from: aws.vpc.bytes, to: network.bytes, type: long}
+
+  {{ if .community_id }}
```

Comment:
I'm not sure why we ever made this conditional in the past. I wouldn't do it here or in the future.

Reply:
Will do!